tidy3d/components/medium.py

```diff
@@ -1107,6 +1131,12 @@ def from_nk(cls, n: float, k: float, freq: float, **kwargs):
             medium containing the corresponding ``permittivity`` and ``conductivity``.
         """
         eps, sigma = AbstractMedium.nk_to_eps_sigma(n, k, freq)
+        if eps < 1:
+            raise ValidationError(
+                "Dispersiveless medium must have 'permittivity>=1`. "
+                "Please use 'Lorentz.from_nk()' to covert to a Lorentz medium, "
+                "or 'AbstractMedium.from_general_nk() to automate the process."
```

Comment:
they might not know what `AbstractMedium` is, you could also phrase this as `or use the '.from_general_nk()' method in all mediums.`.